src/components/Card.js

```diff
@@ -38,13 +60,15 @@ class Card {
     this._cardImage = this._cardElement.querySelector(".card__image");
     this._cardTitle = this._cardElement.querySelector(".card__title");
     this._likeButton = this._cardElement.querySelector(".card__like-button");
+    this._likeCounter = this._cardElement.querySelector(".card__like-counter");
```

Comment:
Good job that you've found these variables only once here and defined them as **class fields**